data/entities.yaml

```diff
@@ -81,6 +90,14 @@
     be found by drilling beneath the surface.
```

Comment:
I guess this comment about finding additional copper beneath the surface is not true, right?  We should change it to say that you can find additional copper by drilling beneath the mountains.

Reply:
Fixed.